zerver/migrations/0143_reactions_realm_emoji_name_to_id.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+import ujson
+
+from collections import defaultdict
+from django.conf import settings
+from django.db import migrations, models
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from typing import Any, Dict
+
+def realm_emoji_name_to_id(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Reaction = apps.get_model('zerver', 'Reaction')
+    RealmEmoji = apps.get_model('zerver', 'RealmEmoji')
+    realm_emoji_by_realm_id = defaultdict(dict)   # type: Dict[int, Dict[str, Any]]
+    for realm_emoji in RealmEmoji.objects.all():
+        realm_emoji_by_realm_id[realm_emoji.realm.id] = {
```

Comment:
I would spell this `.realm_id` -- it'll potentially save a database query.

(When you have a foreign key in Django, `.realm_id` will always be the same as `.realm.id`, but may save going to the DB.

Reply:
Okay, got it! Great! :)

Reply:
Changed and pushed, PTAL! Thanks!